sklearn/model_selection/_search.py

```diff
@@ -271,9 +292,11 @@ def __iter__(self):
                 yield param_grid[i]
 
         else:
-            # Always sort the keys of a dictionary, for reproducibility
-            items = sorted(self.param_distributions.items())
             for _ in range(self.n_iter):
+                dist = self.param_distributions[
+                    rnd.randint(len(self.param_distributions))]
```

Comment:
This is an awkwardly numpy way of expressing random.choose

Reply:
fixed